applications/StructuralMechanicsApplication/structural_mechanics_application.h

```diff
@@ -370,6 +373,11 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) KratosStructuralMechanicsAppl
     const SmallDisplacementBbar mSmallDisplacementBbar2D4N;
     const SmallDisplacementBbar mSmallDisplacementBbar3D8N;
 
+    const SmallDisplacementMixedVolumetricStrainElement mSmallDisplacementMixedVolumetricStrainElement2D3N;
```

Comment:
My question is about the missing geometries, I understand that for non linear geometries the volumetric strain does not corresponds with the standard nodes. In any case the prism is missing (which is linear)